OpenDreamRuntime/Objects/Types/DreamObjectAtom.cs

```diff
@@ -38,6 +31,12 @@ protected override void HandleDeletion(bool possiblyThreaded) {
 
         base.HandleDeletion(possiblyThreaded);
     }
+    public string GetDesc() {
+        if (!TryGetVariable("desc", out DreamValue descVar) || !descVar.TryGetValueAsString(out string? desc))
+            return ObjectDefinition.Type.ToString();
```

Comment:
`ObjectDefinition.Type` is already a string.
```suggestion
            return ObjectDefinition.Type;
```